File: agri_dms/report/slow_moving_stock/slow_moving_stock.py
import frappe
def execute(filters=None):
	columns = [{'label':'Distributor','fieldname':'distributor','width':160}, {'label':'Machine','fieldname':'machine_name','width':150}, {'label':'Days in Stock','fieldname':'days','fieldtype':'Int'}]
	# Dummy logic for example
	data = []
	return columns, data

File: agri_dms/agri_dms/report/slow_moving_stock/slow_moving_stock.js
frappe.query_reports["Slow-Moving Stock"] = {
	"filters": [
		{
			"fieldname": "distributor",
			"label": __("Distributor"),
			"fieldtype": "Link",
			"options": "Distributor"
		},
		{
			"fieldname": "threshold_days",
			"label": __("Threshold Days (In Stock For)"),
			"fieldtype": "Int",
			"default": 30
		}
	]
};
